.github/workflows/observer-cd.yml

```diff
@@ -0,0 +1,45 @@
+name: Observer CD
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - staging
+      - trying
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - master
+      - 'auto-cadence-upgrade/**'
+      - 'feature/**'
+      - 'v[0-9]+.[0-9]+'
+
+jobs:
+  docker-push:
+    name: Push to container registry
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.17'
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Build relic
+      run: make crypto_setup_gopath
+    # Provide Google Service Account credentials to Github Action, allowing interaction with the Google Container Registry
+    # Logging in as github-actions@dl-flow.iam.gserviceaccount.com
+    #- name: Docker login
+    #  uses: docker/login-action@v1
+    #  with:
+    #    registry: gcr.io
+    #    username: _json_key
+    #    password: ${{ secrets.GCR_SERVICE_KEY }}
+    - name: Docker build
+      run: make docker-build-flow
+    - name: Docker push
+      run: bash -c 'make docker-push-flow || true'
```

Comment:
is this different than `cd.yml`?

Reply:
Thank you for the review. The feedback was important, thus we included in the PR.
Removed. We anticipated a separate repo for the observer originally.